modules/nw-rfhe-installing-operator-web-console.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * monitoring/using-rfhe.adoc
+
+:_content-type: PROCEDURE
+[id="nw-rfhe-installing-operator-web-console_{context}"]
+= Installing the Baremetal Hardware Event Proxy Operator using the web console
+
+As a cluster administrator, you can install the Baremetal Hardware Event Proxy Operator using the web console.
+
+.Prerequisites
+
+* A cluster installed on bare-metal hardware with nodes that have Redfish hardware.
```

Comment:
Flagging this sentence for inconsistent use of "bare-metal". In other parts of Redfish doc you have used "baremetal".